src/main/api/2/studio-api-2.yaml

```diff
@@ -9057,6 +9057,41 @@ components:
         - size
         - publishingTarget
 
+    DashboardPublishRequestItem:
```

Comment:
Is it possible to return `DashboardPublishingPackage`? If so, let's do that.

Reply:
Please put the `submissioncomment` field as the `label` for now because `label` seems to be always `null`.

Reply:
> Is it possible to return DashboardPublishingPackage? If so, let's do that.

=> Yes, it was the wrong design when I sent the items instead of packages to the response. Fixed to use 
`DashboardPublishingPackage`

> Please put the submissioncomment field as the label for now because label seems to be always null.

=> I added a field `comment` to put `submissioncomment` into. `label` seems to have different meaning, let me know if we want to remove the `label` from the response!